src/poliastro/tests/test_bodies.py

```diff
@@ -412,3 +427,5 @@ def test_Moon(self, set_epoch):
 
         for i, j in zip(value_from_poliastro, value_from_spice):
             assert_quantity_allclose(i, j)
+
+    os.remove(kernel_name)
```

Comment:
This line should be in some teardown method or similar?

Reply:
I removed the line, as removal of the downloaded kernel is not needed.